Assets/Scripts/KeyBindings/StoreKeyBindings.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.IO;
+using Newtonsoft.Json;
+
+
+public static class StoreKeyBindings
+{
+    private const string custom_path = "keybindings_custom.json";
+    private const string default_path = "keybindings_default.json";
+
+    static StoreKeyBindings()
+    {
+      
```

Comment:
You do not need that static initializer, it does nothing, it seems

Reply:
thats true, but it doesn't disrupt anything as well